server-go/main.go

```diff
@@ -105,7 +106,7 @@ type chatRequest struct {
 //   - chat: string
 //   - history: []
 //
-// Returns a JSON payload containing the model response with the following format.
+// Sends a JSON payload containing the model response to the client with the following format.
```

Comment:
@pcoet 
w.r.t https://github.com/google-gemini/example-chat-app/pull/17#discussion_r1789135716
That's right. It doesn't return anything. By return, I meant the API response. I have modified it. Hope its clear